OpenUpSA/wazimap-ng-ui
# fetching indicator data using the new endpoint

## Description
* removing children indicator data from `all_details` API endpoint
* fetching the children indicator data using the new endpoint `children-indicators`
* removing webflow interactions from data mapper and managing expanding/collapsing the menu elements by javascript
* mapchip & data mapper goes into loading state when navigating between geographies

## Related Issue
https://wazimap.atlassian.net/browse/WNCM-191

## How is it tested automatically?
added test steps into `__tests__/gui/data_mapper/data_mapper.js`

## How should a reviewer test it locally
* confirm that the data mapper works the same way it used to worked
* confirm that choropleth & filtering works correctly
* confirm that rich data charts & filtering works correctly
* confirm that multi-version structure is not broken by testing against SANEF profile
* confirm that choropleth is not broken when navigating between geographies


## Screenshots
![image](https://user-images.githubusercontent.com/53019884/151451337-12c4594b-dd1c-4d70-a574-f84d9d38c17c.png)


## Changelog

### Added

### Updated
* `src/js/controller.js` & `src/js/setup/choropleth.js` to use the `data` property of `children-indicators` instead of the `child_data` property of `all_details` when displaying choropleth
* `src/js/elements/mapchip/legend.js` & `src/js/elements/mapchip/mapchip.js` for the mapchip to go into loading state when navigating between geographies
* `src/js/elements/menu.js` to adapt webflow changes to remove webflow interactions from data mapper elements

### Removed


## Checklist


### Pull Request


### Code Quality


### Testing
